scripts/observe_db_clear.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Let's use the existing script db_clear.sh for this purpose.
db_clear.sh is also copied in the knoxautopolicy dockerfile.